packages/cli/src/util/errors-ts.ts

```diff
@@ -59,7 +59,7 @@ export class TeamDeleted extends NowError<'TEAM_DELETED', {}> {
   constructor() {
     super({
       code: 'TEAM_DELETED',
-      message: `Your team was deleted. You can switch to a different one using ${getCommandName(
+      message: `Your team was deleted, or you were removed from the team. You can switch to a different one using ${getCommandName(
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):**

```suggestion
      message: `Your team was deleted or you were removed from the team. You can switch to a different one using ${getCommandName(
```